kodilla-stream/src/main/java/com/kodilla/stream/homework/ForumStats.java

```diff
@@ -0,0 +1,24 @@
+package com.kodilla.stream.homework;
+
+import com.kodilla.stream.User;
+import com.kodilla.stream.UsersRepository;
+
+public class ForumStats {
+    public static void main(String[] args) {
+        double averageNumberOfPostsWhenUserAgeIsUp40 = UsersRepository.getUsersList()
+            .stream()
+            .filter(a -> a.getAge() >= 40 && a.getNumberOfPosts() !=0)
+            .mapToInt(User::getNumberOfPosts)
+            .average()
+            .getAsDouble();
```

Comment:
lepiej `.orElse(0)`

Reply:
a to Typ Optional mogę używać w streamie? zamiast czego ma być .orElse(0)?

Reply:
zamiast getAsDouble() lepiej uzyc orElse(0). To sie wywali, gdy stream bedzie pusty.